### PR TITLE
Another slew of PEP8/flake8 fixes in /internal - more to come

### PR DESCRIPTION
**Pull request by stephendonner**
This isn't complete - I've taken a break (just for now) with a few of the remaining files, and I've left a couple unused variable/wrong variable assignment warnings alone.

@pmeenan r?

### CODE REVIEW

File: internal/blackbox_android.py [hunk 1]
```diff
@@ -119,18 +119,19 @@ def run_task(self, task):
...
-                cmd = 'am start -n {0} -a android.intent.action.VIEW -d "{1}"'.format(activity, \
-                        command['target'].replace('"', '%22'))
+                cmd = 'am start -n {0} -a android.intent.action.VIEW \
+                       -d "{1}"'.format(activity,
+                                        command['target'].replace('"', '%22'))
                 local_intent = os.path.join(task['dir'], 'wpt_intent.sh')
                 remote_intent = '/data/local/tmp/wpt_intent.sh'
                 self.adb.shell(['rm', remote_intent])
-                with open(local_intent, 'wb') as f_out:
+            with open(local_intent, 'wb') as f_out:
```
Comment (pmeenan) [inline internal/blackbox_android.py]: Why did this move out if the if block?

Comment (stephendonner) [reply internal/blackbox_android.py]: Sorry, over-eager "fix" of a spurious indentation warning which actually arose from another PEP8 fix.

Leaving this last warning in unless you want me to delete that variable and its assignment:

```
diff --git a/internal/blackbox_android.py b/internal/blackbox_android.py
index 0ceaa60..8d01278 100644
--- a/internal/blackbox_android.py
+++ b/internal/blackbox_android.py
@@ -125,7 +125,7 @@ class BlackBoxAndroid(AndroidBrowser):
                 local_intent = os.path.join(task['dir'], 'wpt_intent.sh')
                 remote_intent = '/data/local/tmp/wpt_intent.sh'
                 self.adb.shell(['rm', remote_intent])
-            with open(local_intent, 'wb') as f_out:
+                with open(local_intent, 'wb') as f_out:
                     f_out.write(cmd)
             if self.adb.adb(['push', local_intent, remote_intent]):
                 os.remove(local_intent)
Stephens-MacBook-Pro:internal stephendonner$ flake8 blackbox_android.py 
blackbox_android.py:180:13: F841 local variable 'modified' is assigned to but never used
````

**Comment by stephendonner** [comment]
@pmeenan fixed

File: internal/blackbox_android.py [hunk 2]
```diff
@@ -119,18 +119,19 @@ def run_task(self, task):
...
                 remote_intent = '/data/local/tmp/wpt_intent.sh'
                 self.adb.shell(['rm', remote_intent])
                 with open(local_intent, 'wb') as f_out:
                     f_out.write(cmd)
-                if self.adb.adb(['push', local_intent, remote_intent]):
-                    os.remove(local_intent)
-                    self.adb.shell(['chmod', '777', remote_intent])
-                    self.adb.shell([remote_intent])
-                    self.adb.shell(['rm', remote_intent])
+            if self.adb.adb(['push', local_intent, remote_intent]):
```
Comment (pmeenan) [inline internal/blackbox_android.py]: Sorry, I wasn't clear.  It was the "with" but also this "if" block and it's contents also changed indentation.

**Comment by stephendonner** [comment]
No, the fault is of course mine; fixed, now:

https://github.com/WPO-Foundation/wptagent/blob/e5ddd7f91bef6889689d3196a5b21852ce874f4b/internal/blackbox_android.py#L126-L140

**Comment by pmeenan** [comment]
LGTM, thanks

**Comment by stephendonner** [comment]
@pmeenan thanks for your patience and quick, courteous, and always helpful responses, too!

**Pull request merged**